CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
+## 0.7.2
+
+### Changes
+
+-   Minor update to the Accordion story for Storybook.
```

Comment:
I haven't been adding story updates to the CHANGELOG, since Crystal and I figured it was more helpful to keep the CHANGELOG "clean" with API relevant stuff—do you have an opinion on that?

Reply:
Yeah, that makes sense. I'll remove it.

Reply:
If we can get this PR and #291 merged I can run the release for 0.7.2 today.

Reply:
Merging this now.